lib/build.api.js

```diff
@@ -136,23 +153,27 @@ class Build {
           if (key !== 'root' && key !== 'contrib') {
             links.push({
               src: project.config.paths.modules[key],
-              dest: 'sites/all/modules/' + key,
+              dest: path.join(versionDestPaths.modules, key),
               type: 'dir'
             });
           }
         });
 
-        // Add custom themes link.
-        links.push({
-          src: project.config.paths.themes.custom,
-          dest: 'sites/all/themes/custom',
-          type: 'dir'
+        // Add themes links.
+        Object.keys(project.config.paths.themes).forEach((key) => {
```

Comment:
As far as I know there is only one custom theme directory... does this need to be in a loop?